src/Form/test-cases/steam_checkout.html

```diff
@@ -184,7 +184,7 @@ <h3 id="payment_header" class="tab_header">Payment method</h3>
                                             <div class="dselect_container" id="expiration_month_dselect_container"
                                                 style="display: inline-block;">
                                                 <input id="expiration_month" type="hidden" name="expiration_month"
-                                                    value="" data-manual-scoring="expirationMonth">
+                                                    value="">
```

Comment:
These are `type=hidden` and we shouldn't autofill them, so the correct scoring here is no scoring. We ignore them. This is appropriate because even if we filled them, this wouldn't show for the user who would then go and change the values manually in a custom `select` widget.

We don't autofill (neither does Safari), but that's fine in this case.

Reply:
OK, thanks for the clarification.